src/components/calculator.js

```diff
@@ -1,31 +1,53 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
- [ ] Optional) It is a convention to name the file of components in CamelCase just the way you would name your React component. This helps other developers who have a look at your code to distinguish between components and logic files in your root directory. 👍